tasmota/tasmota_xdrv_driver/xdrv_05_irremote_full.ino

```diff
@@ -378,6 +417,16 @@ bool strToBool(class JsonParserToken token, bool def) {
   }
 }
 
+int16_t timeStrToMinutesInt(const JsonParserToken& token, int16_t def) {
+  std::tm t = {};
+  if (token.isStr() && strptime(token.getStr(), "%H:%M", &t)) {
```

Comment:
String literals must be pushed in Flash with `PSTR("%H:%M")` but must be accessed only with 32 bits aligned reads on ESP8266. I'm not sure that `strptime()` is compatible with string literal in Flash.

Can you assess the flash impact of using `strptime()`? I'm concerned that it brings a lot of useless code. For ESP32 it's ok, for ESP8266 it's not.

Reply:
Thanks, good catch! I'll see what it does to flash size and report back.
TBH, I started to wonder if it really makes sense keeping it. I added for symmetry (the output is `HH:MM` so I figured it'd be nice to be able to have input in this format...), not really considering device limitations.
If it consumes a lot of flash, I technically could add a "manual" code doing string split by colon and conversion... but it smells of needless complexity as well...

----
tl;dr; Now that I think about it... it might be an overkill to add in the first place (and int input in minutes might be enough). Thoughts?

Reply:
Keep it as simple as possible. ESP8266 is **REALLY** limited in resources, every byte counts. If input in minutes is ok, leave it simple. Conversion from more complex formats must be done where you have more resources (like MQTT gateway).

Reply:
Now removed alltogether.

FYI the flash impact was:

* with **strptime**
```
RAM:   [=====     ]  50.0% (used 40932 bytes from 81920 bytes)
Flash: [======    ]  64.7% (used 662580 bytes from 1023984 bytes)
```

* without
```
RAM:   [=====     ]  48.9% (used 40096 bytes from 81920 bytes)
Flash: [======    ]  64.4% (used 659096 bytes from 1023984 bytes)
```

... so saved ~3kB of Flash and ~800B of RAM (well worth it! thx for catching it)

Reply:
Indeed it is not worth the cost